app/org/sagebionetworks/bridge/models/UserSession.java

```diff
@@ -20,6 +21,12 @@ public String getSessionToken() {
     public void setSessionToken(String sessionToken) {
         this.sessionToken = sessionToken;
     }
+    public String getSessionId() {
```

Comment:
This is exactly the same as sessionToken, is it not? Both are session-scoped GUIDs. Why two?